sql/migrations/20170113181857_addConstraintsToPeers.sql

```diff
@@ -0,0 +1,11 @@
+/* Add constraints to improve upserts
+ *
+ */
+
+BEGIN;
+
+ALTER TABLE peers
+  ADD CONSTRAINT unique_address
+UNIQUE (ip, port);
```

Comment:
`ALTER TABLE peers ADD CONSTRAINT unique_address USING INDEX peers_unique;` - should be fine, there shouldn't be noticeable performance hit if any.